SmartDeviceLinkTests/RPCSpecs/PayloadSpecs/SDLSecurityQueryPayloadSpec.m

```diff
@@ -0,0 +1,79 @@
+//
+//  SDLsss.m
+//  SmartDeviceLinkTests
+//
+//  Created by Frank Elias on 8/12/21.
+//  Copyright © 2021 smartdevicelink. All rights reserved.
+//
+
+
+#import <Foundation/Foundation.h>
+
+#import <Quick/Quick.h>
+#import <Nimble/Nimble.h>
+
+#import "SDLRPCParameterNames.h"
+#import "SDLRPCFunctionNames.h"
+#import "SDLSecurityQueryPayload.h"
+
+QuickSpecBegin(SDLSecurityQueryPayloadSpec)
+
+__block SDLSecurityQueryPayload* testPayload;
+__block NSDictionary* dict = @{SDLRPCParameterNameResponse:
+                                   @{SDLRPCParameterNameParameters:@{},
+                                     SDLRPCParameterNameOperationName:SDLRPCFunctionNameDeleteCommand}};
+
+NSData* (^testData)(void) = ^NSData* {
+    NSData* jsonData = [NSJSONSerialization dataWithJSONObject:dict options:0 error:0];
+    NSData* binaryData = [NSData dataWithBytes:"PrimitiveString" length:strlen("PrimitiveString")];
```

Comment:
We don't want to test an RPC like this. We want to test something like what we'd really see with JSON string and error code and binary data like we'll see.